node/src/Unison/Runtime/ResourcePool.hs

```diff
@@ -36,8 +35,8 @@ addResourceToMap release pool p r wait maxPoolSize getThread poolSizeR queue = d
       >> STM.atomically (TQ.writeTQueue queue key)
     else release r
 
-_acquire :: (Ord p, H.Hashable p) => (p -> IO r) -> (r -> IO ()) -> Cache p r -> MaxPoolSize -> IORef.IORef Int -> ReaperQueue p -> IO CC.ThreadId -> p -> Int -> IO (r, IO ())
-_acquire acquire release pool maxPoolSize poolSizeR queue getThread p wait = do
+_acquire :: (Ord p, H.Hashable p) => (p -> IO r) -> (r -> IO ()) -> Cache p r -> Int -> MaxPoolSize -> IORef.IORef Int -> ReaperQueue p -> IO CC.ThreadId -> p -> IO (r, IO ())
```

Comment:
What are the units on that `Int` used for the `wait` parameter? At the very least, name the parameter something like `waitTimeInSeconds` or whatever the units are. But a little better would be to introduce a type alias like `type Seconds = Int` (newtype probably overkill here) and then reference it in the signature, which becomes easier to scan. You could also introduce `type PoolSize = Int` and use an `IORef PoolSize` as well. I am sort of ambivalent on that.